### PR TITLE
Add ability to push images with multiple tags

### PR DESCRIPTION
**Pull request by swetharepakula**
* Add one more tag for e2e-test images

/assign @gauravkghildiyal

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ steps:
   env:
   - DOCKER_CLI_EXPERIMENTAL=enabled
   - REGISTRY=gcr.io/k8s-ingress-image-push
-  - VERSION=$_PULL_BASE_REF
+  - VERSION="${_PULL_BASE_REF} infrastructure-public-image-${_PULL_BASE_REF}"
```
Comment (gauravkghildiyal) [inline cloudbuild.yaml:29]: nit: VERSIONS ?

Comment (swetharepakula) [reply cloudbuild.yaml:29]: I am a little worried about changing the name of the variable incase there are other scripts built on top of push-multiarch.sh that we are unaware about.

By reusing `VERSION` and handling both single version or a list felt like a safer option. What do you think?

Comment (gauravkghildiyal) [reply cloudbuild.yaml:29]: Yeah I can relate to this. Sure let's keep things as they are.

**Comment by swetharepakula** [comment]
/hold remove 

Script has been tested

**Comment by gauravkghildiyal** [comment]
/lgtm

Thanks!

**Pull request merged**